src/components/Account/ScreenProviders.tsx

```diff
@@ -110,25 +129,21 @@ function ProviderButton({ id, provider, onActivate }) {
         }
       `}
     >
-      <img src={provider.image} alt="" height={5.25 * GU} />
-      <div
-        css={`
-          margin-top: ${1 * GU}px;
-          ${textStyle('body1')};
-        `}
-      >
-        {provider.name}
-      </div>
+      {provider && (
+        <>
+          <img src={provider.image} alt="" height={5.25 * GU} />
+          <div
+            css={`
+              margin-top: ${1 * GU}px;
+              ${textStyle('body1')};
+            `}
+          >
+            {provider.name}
+          </div>
+        </>
+      )}
```

Comment:
Provider now can be undefined, here i choose just not show anything while provider its not defined.

Using 'if else' with message for example "Provider not found" could lead to flickering the UI.